app/src/main/java/com/example/footprnt/Profile/EditPost.java

```diff
@@ -120,6 +100,50 @@ public void done(ParseException e) {
             }
         });
 
+
+        mBtnSave.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                post.setTitle(mEtTitle.getText().toString());
```

Comment:
if mEtTitle is empty does mEtTitle.getText() return null? Could this cause a NullPointerException here?